includes/class-wcpdf-settings-callbacks.php

```diff
@@ -382,6 +382,15 @@ public function media_upload( $args ) {
 			if ( !empty($attachment_height) && !empty($in_height) ) {
 				$attachment_resolution = round(absint($attachment_height)/$in_height);
 				printf('<div class="attachment-resolution"><p class="description">%s: %sdpi</p></div>', __('Image resolution','woocommerce-pdf-invoices-packing-slips'), $attachment_resolution );
+
+				// warn the user if the image is unnecessarily large
+				if ($attachment_resolution > 600 ) {
+					echo 
+					"<p class='notice notice-warning inline is-dismissible'>" 
```

Comment:
this is not really dismissible so it shouldn't get that class!

Reply:
(also discussed [here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/219#discussion_r735905901), which you marked as resolved)